Sources/MapboxCoreNavigation/PassiveNavigationEventDetails.swift

```diff
@@ -58,7 +58,7 @@ struct PassiveNavigationEventDetails: NavigationEventDetails {
         try container.encodeIfPresent(coordinate?.longitude, forKey: .longitude)
         try container.encodeIfPresent(userIdentifier, forKey: .userIdentifier)
         try container.encodeIfPresent(appMetadata, forKey: .appMetadata)
-        try container.encodeIfPresent(feedbackType?.description, forKey: .feedbackType)
+        try container.encodeIfPresent(feedbackType?.typeKey, forKey: .feedbackType)
```

Comment:
Do we really need `feedbackType` here? I see that we add type and subtype manually to the resulting dictionary. Or maybe it's better to use these coding keys in that place to prevent hardcoded keys.